### DISCUSSION TITLE
No concurency limits debug message is unimportant

### DISCUSSION THREAD

**Discussion by ravecat** [renovatebot/renovate#17597] [Request Help]
Catch error [No concurency limits](https://gitlab.com/rvct/structures-algorithms/-/jobs/2975401508#L1025). Resolved config [available](https://gitlab.com/rvct/structures-algorithms/-/jobs/2975401508#L390) in the same logs

Meantime similar command run [successfully](https://gitlab.com/rvct/templates/-/jobs/2975379875) for another repo

Could you clarify that case? Didn't find any info in docs and issues

**Comment by rarkins** [comment]
It's not an error, it's a debug

**Reply by ravecat** [reply]
@rarkins yes I undesrstand and tried with another repo, got same ["debug" info](https://gitlab.com/rvct/mendeleev/-/jobs/2976673467) without any renovatebot activity.

**Reply by viceice** [reply]
that's only a new debug message, ignore it. it's just s message for http concurrency

**Reply by ravecat** [reply]
@rarkins  Could you explain your negative reaction, I try to understand why renovatebot haven't activity on repos, log inside renovatebot image hasn't enough info for me

**Reply by ravecat** [reply]
@viceice ok, why doesn't the renovatebot make any MR and return http concurrency log? maybe I don't understand something

**Reply by rarkins** [reply]
This discussion has an answer - the message you asked about is unimportant

**Reply by viceice** [reply]
that message is irrelevant for your question. check the packagefiles with update debug message
